### ISSUE TITLE
Postfix issues

### ISSUE THREAD

**Issue by bnjbvr** [cozy-labs/cozy-docker#15]
I can't seem to be able to send any email from my docker hosted cozy. Adding information as I find them:
- `rsyslog` was stopped, had to start it by hand. Not sure if I did actually stop it once, or if it's set up to not start on a fresh instance. Not related to sending mail, but related to how to debug them :D
- the chrooted environment is missing some links, for a reason I ignore: `/var/spool/postfix/etc` should contain a copy of `/etc/services` and `/etc/resolv.conf`

**Comment by Kloadut** [comment]
Ok, so I assume your PR is fixing this issue ? https://github.com/cozy-labs/cozy-docker/pull/16

**Comment by bnjbvr** [comment]
Yes it does! (at least for the postfix bits, not the rsyslog not being started bits)

**Comment by Kloadut** [comment]
Okay ! I'll test this right away

**Comment by bnjbvr** [comment]
Anyhow, it seems the configuration of the postfix instance was done in a weird way, as it doesn't have any files inside the /etc chroot. I've seen on ubuntu that when installing, if you chose the first option when configuring postfix for the first time (there's an ncurses interface showing up, it might be skipped on docker), it wasn't copying anything inside the chroot. Could that be the issue?
